sdk/storage/src/blobs/blob_client.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "blobs/blob_client.hpp"
+
+#include "common/storage_common.hpp"
+#include "http/curl/curl.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  BlobClient BlobClient::FromConnectionString(
+      const std::string& connectionString,
+      const std::string& containerName,
+      const std::string& blobName,
+      const BlobClientOptions& options)
+  {
+    auto parsedConnectionString = ParseConnectionString(connectionString);
+
+    std::string accountName;
+    std::string accountKey;
+    std::string blobEndpoint;
+    std::string EndpointSuffix;
+    std::string defaultEndpointsProtocol = ".core.windows.net";
```

Comment:
>".core.windows.net" [](start = 43, length = 19)

Minor: This should either be in protocol layer or Common namespace.

Reply:
We are going to move these string literals into a separate file in the future.